docker/docker-init.sh

```diff
@@ -15,12 +15,10 @@ then
     echo "${SSH_PRIVATE_KEY}" | tr -d '\r' | ssh-add -
 fi
 
-# space-separated list of balena CLI commands (filled in through `sed`
-# in a Dockerfile RUN instruction)
-CLI_CMDS="help"
-
-# try to determine if command is a balena CLI command
-if [[ " ${CLI_CMDS} " =~ " ${1} " ]]
```

Comment:
Shellcheck wasn't happy with this line. It seemed to work anyway when I tested it though?
https://github.com/koalaman/shellcheck/wiki/SC2076

Reply:
> SC2076: _it'll match literally rather than as a regex_

That's how I meant it to work, nothing wrong with it. Removing the quotes as per their advice would introduce a bug.

Reply:
That's what I thought, odd behaviour from shellcheck.